audtorch/datasets/emodb.py

```diff
@@ -0,0 +1,62 @@
+import os
+import glob
+
+from typing import Callable
+
+from audtorch.datasets.base import AudioDataset
+from audtorch.datasets.utils import download_url
+
+
+__doctest_skip__ = ['*']
+
+
+class EmoDB(AudioDataset):
+    r"""EmoDB data set.
+
+    Open and publicly available data set of acted emotions:
+    http://www.emodb.bilderbar.info/navi.html
+
+    EmoDB is a small audio data set collected in an anechoic chamber in the
+    Technical University of Berlin, it contains 5 male and 5 female speakers,
```

Comment:
It's a little bit strange, but the official name of the University is `Technische Universität Berlin` or `Berlin Institute of Technology`. So please pick one of those.